core/src/test/java/org/apache/accumulo/core/clientImpl/ThriftTransportKeyTest.java

```diff
@@ -136,8 +138,8 @@ public void testSimpleEquivalence() {
 
     replay(clientCtx);
 
-    ThriftTransportKey ttk =
-        new ThriftTransportKey(HostAndPort.fromParts("localhost", 9999), 120_000, clientCtx);
+    ThriftTransportKey ttk = new ThriftTransportKey(ThriftClientTypes.CLIENT,
+        HostAndPort.fromParts("localhost", 9999), 120_000, clientCtx);
 
     assertEquals(ttk, ttk, "Normal ThriftTransportKey doesn't equal itself");
```

Comment:
This is a preexisting problem with this test, would be nice to make it follow the pattern in the other existing test.

```suggestion
    assertEquals(ttk, ttk, "Normal ThriftTransportKey doesn't equal itself");
    assertEquals(ttk.hashCode(), ttk.hashCode());
```

Reply:
Added this in 1a3d5e0